### PR TITLE
Add extensions support for phony validator

### PR DESCRIPTION
**Pull request by Kukunin**

### CODE REVIEW

File: lib/validators/phony_validator.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@ def validate_each(record, attribute, value)
 
     @record = record
     value = PhonyRails.normalize_number(value.dup, default_country_code: normalized_country_code) if normalized_country_code
+    value, = PhonyRails.extract_extension(value)
```
Comment (joost) [inline lib/validators/phony_validator.rb]: What is the comma doing here?

**Comment by joost** [comment]
@Kukunin would be happy to merge. Please describe how it works in README.

**Comment by joost** [comment]
@Kukunin closing this. Reopen if you want it in.

File: lib/validators/phony_validator.rb (hunk 1 above)
Comment (Kukunin) [inline lib/validators/phony_validator.rb]: it's equivalent to `value, _ = PhonyRails.extract_extension(value)`. may look creepy though

**Pull request closed without merging**